src/map/status.c

```diff
@@ -6112,6 +6120,11 @@ static short status_calc_aspd_rate(struct block_list *bl, struct status_change *
 		aspd_rate += sc->data[SC_STEAMPACK]->val2 * 10;
 	if (sc->data[SC_SKF_ASPD] != NULL)
 		aspd_rate -= sc->data[SC_SKF_ASPD]->val1 * 10;
+	//TODO: test the aspd inc
+#ifdef RENEWAL
+	if (sc->data[SC_INC_AGI])
+		aspd_rate += sc->data[SC_INC_AGI]->val1 * 10;
+#endif
```

Comment:
The formula is not working properly.
I Have changed to (under the aspd bonus):

```
		if (sc->data[SC_INCREASEAGI])
			bonus += sc->data[SC_INC_AGI]->val1;
```
in the bottom of the section `**// ASPD percentage values**`